Core/include/Acts/Surfaces/Surface.hpp

```diff
@@ -434,35 +430,46 @@ class Surface : public virtual GeometryObject,
   /// extrinsic Euler angles)
   ///
   /// @param gctx The current geometry context object, e.g. alignment
-  /// @param derivatives Path length derivatives of the free, nominal
-  /// parameters to help evaluate change of free track parameters caused by
   /// change of alignment parameters
-  /// @param position The position of the paramters in global
-  /// @param direction The direction of the track
+  /// @param parameters is the free parameters
+  /// @param derivatives is the derivative of free parameters w.r.t. path length
   ///
-  /// @return Derivative of bound track parameters w.r.t. local frame
+  /// @return Derivative of bound local position w.r.t. local frame
   /// alignment parameters
   AlignmentToBoundMatrix alignmentToBoundDerivative(
-      const GeometryContext& gctx, const FreeVector& derivatives,
-      const Vector3D& position, const Vector3D& direction) const;
+      const GeometryContext& gctx, const FreeVector& parameters,
+      const FreeVector& derivatives) const;
```

Comment:
```suggestion
      const FreeVector& pathDerivative) const;
```
To clarify what this should contain. Also, should this not be a `FreeRowVector`?

Reply:
The derivative of free parameters w.r.t. path length is a `FreeVector` and the derivative of path length w.r.t. free parameters is a `FreeRowVector`.